ebos/ecltransmissibility.hh

```diff
@@ -676,9 +676,14 @@ private:
                 // Silently discard as it is not between active cells
                 continue;
 
-            if (low == -1 || high == -1)
-                OPM_THROW(std::logic_error, "NNC between active and inactive cells (" <<
-                          low << " -> " << high);
+            if (low == -1 || high == -1) {
+                // Discard the NNC if it is between active cell and inactive cell
+                std::ostringstream sstr;
+                sstr << "NNC between active and inactive cells ("
+                     << low << " -> " << high << ")";
```

Comment:
the message here is probably not really useful for people who do the deck-wrestling: at least one of `high` and `low` is -1...

Reply:
also: are these variables holding Cartesian indices? As far as I know, people who create decks have no good way to translate compressed indices to the ones which they actually need to use (i.e., Cartesian ones).

Reply:
Agreed, probably we should return I, J, K in order to help to fix the deck.  

At the moment, we will not get any message yet, since we We call `setupLogging();` after `setupEbosSimulator();` in `FlowMainEbos`.